src/Entities/BaseProduct.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace RobotStores\Entities;
+
+class BaseProduct
+{
+    protected int $id;
+    protected string $title;
+    protected float $price;
+    protected string $image;
+
+    public function getID():int {
+        return $this->id;
+    }
+
+    public function getTitle():  string {
```

Comment:
maybe that space moved down here.. there seem to be two before this return type :D

Reply:
drop down curly brace here - and throughout the other files too